onnx/common/ir_pb_converter.cc

```diff
@@ -145,6 +145,18 @@ void convertAttribute(const ONNX_NAMESPACE::AttributeProto& ap, Node* n) {
       n->ts_(sym, std::move(tensors));
       break;
     }
+    case ONNX_NAMESPACE::AttributeProto_AttributeType_TYPE_PROTO:
```

Comment:
I have a PR pending that introduces SparseTensor as first class citizen. Should I also add some changes here so sparse attributes are handled or it is something that the converter team would address later?

Reply:
Yeah would be great if you can add sparse tensor attributes. Thanks!